src/app/modules/gb-export-module/data-table-components/gb-data-table-dimensions/gb-data-table-dimensions.component.ts

```diff
@@ -25,8 +25,8 @@ export class GbDataTableDimensionsComponent implements OnInit {
 
   onChange() {
     this.dataTableService.validateDimensions();
-    this.dataTableService.dataTable.isDirty = true;
-    // TODO: update data table
+    this.dataTableService.isDirty = true;
```

Comment:
setting the flag here is redundant since it is set inside the updateDataTable() method, will change